src/components/pages/farmer-member/list/modals/multi-marker-map/index.tsx

```diff
@@ -34,40 +40,58 @@ const MultiMarkerMapModal = () => {
     queryKey: ["AllFarmerByUnion"],
     queryFn: () => axGetAllFarmerByUnion(5), // unionId is hardcoded
     enabled: fetchData,
-    staleTime: 1000 * 60 * 60 * 24 * 10, // 10 days in milliseconds
+    staleTime: 1000 * 60 * 60 * 24 * 2, // 2 days in milliseconds
     gcTime: 1000 * 60 * 60 * 24 * 20, // 20 days in milliseconds
   });
 
+  const getFarmerCountAndPlots = (selected) => {
+    const totalFarmer = selected.length;
+    let totalPlots = 0;
+    selected.forEach((farmer) => {
+      totalPlots += farmer.location.type === "Point" ? 1 : farmer.location.coordinates.length;
```

Comment:
if there is a possibility of having multiple types of farmer locatiions, can we just have a set of constants in the constants file that we can use it here?